src/routes/+layout.svelte

```diff
@@ -2,12 +2,19 @@
   import "./app.css";
   import Nav from "$lib/Nav.svelte";
   import Footer from "$lib/Footer.svelte";
+  import { page } from "$app/stores"; // access the current page route
+
   export let data;
 </script>
 
 <div class="w-full min-h-screen bg-white">
-  <Nav {data} />
+  {#if $page.url.pathname !== "/signIn"}
+    <Nav {data} />
+  {/if}
+
   <slot />
 
-  <Footer />
```

Comment:
Same for the footer

we cld also look fix the signUp page
something like this 
```
{#if $page.url.pathname !== "/signIn" && $page.url.pathname !== "/signUp"}
  <Footer />
{/if}```